src/processing/uendelig_bakke/uendelig_bakke.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Uendelig bakke
+level: 1
+author: Kine Gjerstad Eide
+language: nb
+---
+
+# Introduksjon {.intro}
+
+I denne oppgava skal vi få underlaget til å bevege seg uendeliglangt bortover. Det skal vi gjøre ved å ha to like bilder ved siden av hverandre som beveger seg til venstre. Med en gang et av bildene er kommet på utsiden av skjermen, så flytter vi det til høyre siden av det andre bildet slik at det ser ut som om vi har et langt bilde som beveger seg.
+
+Når all koden er skrevet ferdig vil du ha begynnelsen av et spill som ser slik ut:
+
+![](preview.png)
+
+# Steg 1: Oppsett {.activity}
+Først må vi sette opp programmet vårt med en `draw `og en `setup`-metode
+
+![](setup.png)
+
+nå kan vi lagre programmet vårt slik at vi kan finne det igjen senere. Trykk `CTRL`+ `s` på tastaturet og gi det et navn som du husker, for eksempel `rulling`.
+
+Nå kan du også høyreklikke på dette bildet av en bakke, og lagre det i samme mappe som spillet ditt ble lagret i. Processing lagrer filer i `.pde` format.
+
+![](bakke.png) 
+
+Så kan vi sette opp størrelsen på vinduet og en bakgrunnsfarge i vinduet.
+
+## Gjør dette {.check}
+- [ ] I `setup`-metoden, altså mellom krøllparentesene etter `setup` skriv: `size(800, 800);`
+- [ ] I `draw`-metoden, skriv: `background(100, 100, 200);`
+
+## Forklaring av innebygde metoder {.try}
+Både `size` og `background` er innebygde metoder, det betyr at noen har skrevet dem ferdig for oss og dermed kan vi bare bruke de når de treng. Når vi bruker metoder kalles å gjøre metodekall.
+
+Tallene i metodekallet på `size` betyr hvor høyt og hvor bredt vinduet skal være.  
+Tallene i `background` bestemer hvor mye rødt, grønt og blått det skal være.  
+Hvert av de tallene kan bare være mellom 0 og 255. 
+
+Alt som står inni parentesene på en metode vi bruker er argumentene, eller parameterne, til metoden. 
+
+## Gjør dette: {.check}
+- [ ] Prøv å endre argumentene til `background` til du finner en farge du liker.
+
+# Steg 2: Hente inn bilde {.activity}
+
+I Processing kan et bilde bli lagret i en variabel av typen `PImage`, og hentes inn ved å bruke metoden `loadImage();`. Denne metoden tar navnet på bildet som et argument, og brukes gjerne inne i `setup`.
+
+![](bilde.png)
```

Comment:
slå heller sammen bilde.png og bilde2.png slik at du ser all koden. ikke vits å dele opp i to. også har du forklaringen under. nå kommer forklaring før selve koden.